modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/DistributedSqlConfiguration.java

```diff
@@ -227,4 +242,16 @@ public int defaultQueryTimeout() {
     public void listenDefaultQueryTimeout(DistributePropertyListener<Integer> lsnr) {
         dfltQueryTimeout.addListener(lsnr);
     }
+
+    /** */
+    public boolean isDisableCreateLuceneIndexForStringValueType() {
+        Boolean ret = disableCreateLuceneIndexForStringValueType.get();
+
+        return ret != null && ret;
+    }
+
+    /** */
+    public GridFutureAdapter<?> createLuceneIndexForStringValueType(boolean create) throws IgniteCheckedException {
```

Comment:
Looks misleading. I expect that we will enable the functionality (when calling with true value), but in fact we are disabling it.